web/modules/custom/eulogin/joinup_eulogin.module

```diff
@@ -108,7 +108,22 @@ function joinup_eulogin_form_user_form_alter(array &$form, FormStateInterface $f
     return;
   }
 
-  $form['account']['mail']['#description'] = t('The email address is not made public and will only be used if you wish to receive certain news or notifications by email.');
+  $form['header'] = [
+    '#weight' => -100,
+    '#type' => 'html_tag',
+    '#tag' => 'h3',
+    '#value' => t('EU Login Account information'),
+  ];
+  $form['header_description'] = [
+    '#weight' => -99,
+    '#type' => 'html_tag',
+    '#tag' => 'span',
+    '#value' => t('Your name and E-mail data is inherited from EU Login. To update this information, you can visit your EU Login account page <a href="https://webgate.ec.europa.eu/cas/eim/external/restricted/edit.cgi" rel="nofollow" target="_blank">here</a>. Synchronisation will take a few minutes and it will be visible the next time you login on Joinup.'),
+  ];
```

Comment:
This is not resolving the separation of "EU Login Account information" block from the rest of the page as it is shown in the ticket's description mockup. The decision was to implement field groups. Also, we have config for service URLs (see the `register_url` key from `joinup_eulogin.settings.yml`. Let's create a similar `edit_url` instead of hardcoding it here.

Reply:
We keep the actual solution as it has been discussed. Probably we need a frontend followup.